src/arcade/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj

```diff
@@ -37,11 +37,10 @@
   <UsingTask TaskName="Microsoft.DotNet.Arcade.Sdk.GenerateChecksums" AssemblyFile="$(ArcadeSdkBuildTasksAssembly)" />
 
   <!--
-    In VMR inner-build copy to local storage, do not push to AzDO.
-    Inner-repo builds outside of VMR do not use arcade publishing.
+    In VMR copy to local storage, do not push to AzDO.
     Define this property before importing repo Publishing.props, to have it control any repo-specific publishing.
   -->
-  <PropertyGroup Condition="('$(DotNetBuildPhase)' == 'InnerRepo' or '$(UseArPowBuildInfra)' == 'false') and '$(DotNetBuildOrchestrator)' == 'true'">
+  <PropertyGroup Condition="'$(DotNetBuildRepo)' == 'true' and '$(DotNetBuildOrchestrator)' == 'true'">
```

Comment:
If we're getting rid of repo source-build and getting rid of usages of "single repo in orchestrated build mode where `DotNetBuildOrchestrator != true`", we can simplify these conditions.

Reply:
We aren't getting of repo source-build. It will still continue to exist just without prebuilt reporting, inner clone and the outer/inner build complexity.